src/Week1/Homework1.hs

```diff
@@ -0,0 +1,55 @@
+module Homework1 where
+
+-- 1
+toDigits :: Integer -> [Integer]
+toDigits num =
+  helper num []
+ where
+  helper :: Integer -> [Integer] -> [Integer]
+  helper n l =
+   if n < 1
+   then l
+   else helper (div n 10) ((mod n 10):l)
```

Comment:
You can use `divMod` function to get division and mod automatically

Reply:
I feel like there should be a better way than this to use divMod
```
   else helper a $ b:l
    where
      (a,b) = divMod n 10
```
am I right?